src/test/java/org/jenkinsci/plugins/workflow/job/ReverseBuildTriggerTest.java

```diff
@@ -47,34 +46,30 @@
 
     @Issue("JENKINS-33971")
     @Test public void upstreamMapRebuilding() throws Exception {
-        story.addStep(new Statement() {
-            @Override public void evaluate() throws Throwable {
-                story.j.jenkins.setQuietPeriod(0);
-                WorkflowJob us = story.j.jenkins.createProject(WorkflowJob.class, "us");
-                us.setDefinition(new CpsFlowDefinition(""));
-                us.addProperty(new SlowToLoad()); // force it to load after ds when we restart
-                WorkflowJob ds = story.j.jenkins.createProject(WorkflowJob.class, "ds");
-                ds.setDefinition(new CpsFlowDefinition(""));
-                ds.addTrigger(new ReverseBuildTrigger("us", Result.SUCCESS));
-                story.j.assertBuildStatusSuccess(us.scheduleBuild2(0));
-                story.j.waitUntilNoActivity();
-                WorkflowRun ds1 = ds.getLastCompletedBuild();
-                assertNotNull(ds1);
-                assertEquals(1, ds1.getNumber());
-            }
+        story.then(r -> {
```

Comment:
[for review](https://github.com/jenkinsci/workflow-job-plugin/pull/63/files?w=1#diff-e91c07a0ee602a64d0e87fb5a3f7e047R49)